app/lib/matador/components/job-info/JobInfoPanel.tsx

```diff
@@ -0,0 +1,70 @@
+import { Divider, Grid, Title } from "@mantine/core";
+import JobInfo from "./JobInfo";
+
+interface JobInfoPanelProps {
```

Comment:
export the type

Reply:
Fixed with 35c6421357701641e9eedeec5819dece439e234c